autotest/cpp/test_gdal_jpegxr.cpp

```diff
@@ -0,0 +1,389 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
My rationale is this:
Test in C++ is more convenient during a driver development, debugging, etc.. So, I aimed at that first. Once a driver offers minimum usability, switch to Python is an obvious path and the C++ test remains 'static', not aimed for active maintenance really.

My plan is:
* Not add any more test cases to C++ test.
* Add Python test, shift or copy test images to `gdrivers/data`
* If it is desired, remove the C++ test.